code/src/cljs/sixsq/nuvla/ui/pages/credentials/views.cljs

```diff
@@ -979,13 +979,17 @@
 
 
 (defn SingleCredential
-  [{:keys [subtype name description] :as credential}]
+  [{:keys [subtype name description id] :as credential}]
   [ui/TableRow
    [ui/TableCell {:floated :left
-                  :width   2}
-    [:span name]]
+                  :width   6}
+    name [:br]
+    [ui/Popup {:trigger (r/as-element [ui/CopyToClipboard {:text id} [:div {:class "credential-id-container"} id]])
```

Comment:
I think it's better to extend the copy to clipboard component that we use in different places in the UI and use it also here. I know it's not the subject of this issue but it's an opportunity to do it. If do you agree merge following PR in your branch before merging yours https://github.com/nuvla/ui/pull/1581/files.